packages/openneuro-app/vite.config.js

```diff
@@ -27,6 +27,10 @@ export default defineConfig({
     port: 80,
     host: "0.0.0.0",
     cors: true,
+    watch: {
+      usePolling: true, // Enable polling for file watching
```

Comment:
This has a pretty significant performance cost if it isn't needed for a large project like this. Might be better to use these overrides on macOS and document it for developer setup instead.

In `.env`:
```
CHOKIDAR_USEPOLLING=true
CHOKIDAR_INTERVAL=1000
```

Reply:
removed this from vite.config and added polling config to docker-compose and config.env